### ISSUE TITLE
Horse Grindstone and Torches

### ISSUE THREAD

**Issue by uglystick76** [Darkosto/SevTech-Ages#1305]
When ever a torch is placed within the 7x7x2 radius of the horse grindstone it causes an invalid placement of the grindstone.  This is not the case with the horse chopper and the horse press.

**Comment by Marioluigi7896** [comment]
![2018-04-02_15 52 22](https://user-images.githubusercontent.com/30518749/38220222-e913c436-368d-11e8-8487-0a37d3b42a61.png)

**Comment by Marioluigi7896** [comment]
This occurs with all modded torches

**Comment by GoryMoon** [comment]
Do you have the chopping block one up? 
They are intended and works best if they are as on the image above.
There is no difference of what torch you use either as only replaceable blocks like grass and plants are allowed to be in the area.

**Comment by 630R63** [comment]
The chopper needs to be one block higher

**Comment by GoryMoon** [comment]
No, the only one that needs to be one higher is the grindstone, all other horse powered blocks can be place on directly on the ground that the horse walks on.
The reason for the grindstone is so the connection point would be the same related to the horse and so that the height would match the height of the horse.
